hardware_interface/include/hardware_interface/actuator_state_interface.h

```diff
@@ -64,22 +65,142 @@ class ActuatorStateHandle
     {
       throw HardwareInterfaceException("Cannot create handle '" + name + "'. Effort data pointer is null.");
     }
+
+  }
+
+  ActuatorStateHandle(const std::string& name, const double* pos, const double* vel, const double* eff,
```

Comment:
New constructors missing function comments. In addition to parameter descriptions, should consider adding brief blurb highlighting the differences between the constructors.

Same applies to `joint_state_interface.h`

Reply:
done